contracts/src/v0.8/llo-feeds/dev/interfaces/IChannelVerifierProxy.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.16;
```

Comment:
I'd suggest leaving the Proxy pragmas more flexible since other users will need to pull these into their contracts.